src/main/scala/xiangshan/cache/dcache/loadpipe/LoadPipe.scala

```diff
@@ -195,7 +196,11 @@ class LoadPipe(id: Int)(implicit p: Parameters) extends DCacheModule with HasPer
 
   io.replace_way.set.valid := RegNext(s0_fire)
   io.replace_way.set.bits := get_idx(s1_vaddr)
-  val s1_repl_way_en = UIntToOH(io.replace_way.way)
+
+  val s1_invalid_vec = wayMap(w => !meta_resp(w).coh.isValid())
+  val s1_have_invalid_way = s1_invalid_vec.asUInt.orR
+  val s1_invalid_way_en = ParallelPriorityMux(s1_invalid_vec.zipWithIndex.map(x => x._1 -> UIntToOH(x._2.U(nWays.W))))
+  val s1_repl_way_en = Mux(s1_have_invalid_way, s1_invalid_way_en, UIntToOH(io.replace_way.way))
```

Comment:
`OHToUInt` is used in L417. We should provide both `s1_repl_way_en_oh` and `s1_repl_way_en_enc`?